vnext/local-cli/generator-windows/templates/cpp/src/App.cpp

```diff
@@ -34,7 +34,7 @@ App::App() noexcept
 #endif
 
     PackageProviders().Append(make<ReactPackageProvider>()); // Includes all modules in this project
-
+    RegisterNativeModulePackages;
```

Comment:
>RegisterNativeModulePackages [](start = 4, length = 28)

Is it supposed to be a function call? Then it is missing parentheses ().

Reply:
It is defined as a macro, which is empty by default.

---
In reply to: [391158416](https://github.com/microsoft/react-native-windows/pull/4293#discussion_r391158416) [](ancestors = 391158416)

Reply:
I see. It is a macro. Could you use all upper case letters with words separated by underscore for macros?
E.g.

REACT_REGISTER_MODULE_PACKAGES

I found that many times when we mask macro as a function we cause unpredicted errors in future. E.g. min and max macro.

Would it be possible to change the macro to a real inline function? Or at least make it looking as a function call?

REACT_REGISTER_MODULE_PACKAGES();


---
In reply to: [391158416](https://github.com/microsoft/react-native-windows/pull/4293#discussion_r391158416) [](ancestors = 391158416)